scripts/make-flavor-spritesheet.js

```diff
@@ -0,0 +1,74 @@
+/* global __dirname process */
+
+// TODO: Reconcile with base project marker assets, and make this tool smart
+// enough to only package the sprites it needs for a given flavor.
+// TODO: Automatically build spritesheets as part of the static build.
+
+require("dotenv").config({ path: "src/.env" });
+
+const fs = require("fs-extra");
+const Spritesmith = require("spritesmith");
+const glob = require("glob");
+const path = require("path");
+const shell = require("shelljs");
+
+const flavorMarkersPath = path.resolve(
+  __dirname,
+  "../src/flavors",
+  process.env.FLAVOR,
+  "static/css/images/markers",
+);
+
+// Remove old spritesheets so they don't get baked into the new spritesheets!
+shell.rm(path.resolve(flavorMarkersPath, "spritesheet.png"));
+shell.rm(path.resolve(flavorMarkersPath, "spritesheet@2x.png"));
+
+const flavorMarkers = glob.sync(
+  path.resolve(flavorMarkersPath, "*.{png,jpg,jpeg,gif}"),
+);
+
+Spritesmith.run({ src: flavorMarkers }, (err, result) => {
```

Comment:
Note that if you are having race condition issues with this script, you can promisify it via:

https://nodejs.org/api/util.html#util_util_promisify_original
or
https://github.com/mapseed/platform/blob/master/scripts/deploy.js#L24

which should make it easier to integrate this script with other scripts.